deployments/ecs/ec2/splunk-otel-collector.json

```diff
@@ -0,0 +1,159 @@
+{
+  "ipcMode": null,
+  "executionRoleArn": null,
+  "containerDefinitions": [
+    {
+      "dnsSearchDomains": null,
+      "environmentFiles": null,
+      "logConfiguration": null,
+      "entryPoint": null,
+      "portMappings": [],
+      "command": null,
+      "linuxParameters": null,
+      "cpu": 0,
+      "environment": [
+        {
+          "name": "ECS_METADATA_EXCLUDED_IMAGES",
+          "value": "[\"quay.io/signalfx/splunk-otel-collector\"]"
+        },
+        {
+          "name": "SPLUNK_ACCESS_TOKEN",
+          "value": "MY_SPLUNK_ACCESS_TOKEN"
+        },
+        {
+          "name": "SPLUNK_CONFIG",
+          "value": "/etc/otel/collector/ecs_ec2_config.yaml"
+        },
+        {
+          "name": "SPLUNK_REALM",
+          "value": "MY_SPLUNK_REALM"
+        }
+      ],
+      "resourceRequirements": null,
+      "ulimits": null,
+      "dnsServers": null,
+      "mountPoints": [
+        {
+          "readOnly": true,
+          "containerPath": "/hostfs",
```

Comment:
Why is this needed?

Reply:
This supposed to be mount point for the host "/" but the volumes below are not correct!

Reply:
Changed to mounting host root to container /hostfs